projects/gloo/api/v1/settings.proto

```diff
@@ -975,10 +975,10 @@ message ConsoleOptions {
 
     // Whether to enable the GraphQL API Explorer. This feature requires a Gloo Edge Enterprise license with GraphQL enabled.
     // Defaults to true.
-    google.protobuf.BoolValue api_explorer_enabled = 2;
+    google.protobuf.BoolValue api_explorer_enabled = 2 [deprecated = true];
 }
 
-// GraphQL settings used by the control plane and UI.
+// Deprecated: GraphQL settings used by the control plane and UI.
 message GraphqlOptions {
```

Comment:
i think there are some other files that we can add the deprecation messages to. 

these are all the files i see 'graphql' references in. we probably don't need to modify the `external/envoy` ones since those are just imported into the user-facing protos
```
projects/gateway/api/v1/virtual_service.proto
projects/gloo/api/external/envoy/extensions/graphql/graphql.proto
projects/gloo/api/external/envoy/extensions/graphql/stitching.proto
projects/gloo/api/v1/enterprise/options/graphql/v1beta1/graphql.proto
projects/gloo/api/v1/options/graphql/graphql.proto
projects/gloo/api/v1/options/service_spec.proto
projects/gloo/api/v1/proxy.proto
projects/gloo/api/v1/settings.proto
```

Reply:
TY, I think something has been off with my VS code search. Updated in a bunch of places. Agree that we don't need to update the external/envoy ones as those docs are going to get deleted as part of an in-flight ticket.